airbyte-integrations/connectors/source-persistiq/source_persistiq/schemas/campaigns.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
Please make sure you only use `null` for nullable type, matching the source API spec. e.g: `id` field is probably not nullable.